extensions/amp-analytics/0.1/variables.js

```diff
@@ -234,20 +234,26 @@ export class VariableService {
     this.register_('SCROLL_LEFT', () =>
       Services.viewportForDoc(this.ampdoc_).getScrollLeft()
     );
-    // Was set async before
-    this.register_('FIRST_CONTENTFUL_PAINT', () => {
-      return Services.performanceFor(
-        this.ampdoc_.win
-      ).getFirstContentfulPaint();
-    });
-
-    this.register_('FIRST_VIEWPORT_READY', () => {
-      return Services.performanceFor(this.ampdoc_.win).getFirstViewportReady();
-    });
 
-    this.register_('MAKE_BODY_VISIBLE', () => {
-      return Services.performanceFor(this.ampdoc_.win).getMakeBodyVisible();
-    });
+    // Call performance-impl.js to get promise returning value for metric
+    this.register_('FIRST_CONTENTFUL_PAINT', () =>
+      Services.performanceFor(this.ampdoc_.win).getMetric('fcp-v')
```

Comment:
Can we make these enum constants somewhere?

Reply:
Are you referring to `FIRST_CONTENTFUL_PAINT` or the metric tick labels? Would that be removed from the final build by the pipeline?

Reply:
The tick labels. Magical strings like this referenced in multiple sources files are begging for a typo to happen.

> Would that be removed from the final build by the pipeline?

Yup. Closure will collapse the `perf.FCP_VISIBLE` into whatever its value is `'fcp-v'`.